lang/ja.yml

```
@@ -83,110 +83,110 @@ INSPECTOR_TOGGLED: "検査官モードが{有効|無効}になりました。"
+PATCH_OUTDATED_1: "{0}は古いデータベースをアップグレードできません。"
+PATCH_OUTDATED_2: "CoreProtect がサポートされているバージョンにアップグレードしてください。"
+PATCH_PROCESSING: "新しいデータを処理しています。しばらくお待ちください..."
+PATCH_SKIP_UPDATE: "{0}の{テーブル|インデックス}の{更新|作成|削除}をスキップします。"
+PATCH_STARTED: "{0} のアップグレードを実行します。しばらくお待ちください..."
+PATCH_SUCCESS: "{0} は正常にアップグレードされました。"
+PATCH_UPGRADING: "データベースのアップグレードが進行中です。しばらくお待ちください..."
+PLEASE_SELECT: "選んでください: \"{0}\" または \"{1}\" 。"
 PREVIEW_CANCELLED: "プレビューがキャンセルされました。"
 PREVIEW_CANCELLING: "プレビューをキャンセルしています..."
 PREVIEW_IN_GAME: "ゲーム内でのみロールバックをプレビューできます。"
-PREVIEW_TRANSACTION: "プレビューできません {コンテナ|在庫} トランザクション。"
+PREVIEW_TRANSACTION: "{コンテナ|在庫}の入れ替えがプレビューできません"
 PURGE_ABORTED: "パージに失敗しました。データベースが破損している可能性があります。"
 PURGE_ERROR: "{0} のデータを処理できません"
-PURGE_FAILED: "パージに失敗しました。後でもう一度やり直してください。"
-PURGE_IN_PROGRESS: "進行中のパージ。後でもう一度やり直してください。"
-PURGE_MINIMUM_TIME: "分離できるのは、より古いデータのみです。({0}{日|時間})"
-PURGE_NOTICE_1: "これには時間がかかる場合がありますのでご注意ください。"
+PURGE_FAILED: "パージに失敗しました。もう一度やり直してください。"
+PURGE_IN_PROGRESS: "パージは進行中です。もう一度やり直してください。"
+PURGE_MINIMUM_TIME: "パージできるのは、より古いデータのみです。({0}{日|時間})"
+PURGE_NOTICE_1: "これには時間がかかる場合があります..."
 PURGE_NOTICE_2: "完了するまでサーバーを再起動しないでください。"
-PURGE_OPTIMIZING: "データベースの最適化。お待ちください..."
+PURGE_OPTIMIZING: "データベースの最適化をしています。しばらくお待ちください..."
 PURGE_PROCESSING: "{0} のデータを処理中..."
 PURGE_REPAIRING: "修復しようとしています。これには時間がかかる場合があります..."
-PURGE_ROWS: "{0}{行|行} のデータが削除されました。"
+PURGE_ROWS: "{0}行のデータが削除されました。"
 PURGE_STARTED: "データの削除は\"{0}\"で開始されました。"
-PURGE_SUCCESS: "データの分離に成功しました。"
-RELOAD_STARTED: "構成の再読み込み-しばらくお待ちください。"
-RELOAD_SUCCESS: "構成が正常に再ロードされました。"
+PURGE_SUCCESS: "データのパージに成功しました。"
+RELOAD_STARTED: "構成の再読み込みをしています。しばらくお待ちください。"
+RELOAD_SUCCESS: "構成が正常に再読み込みされました。"
 ROLLBACK_ABORTED: "ロールバックまたは復元が中止されました。"
-ROLLBACK_CHUNKS_FOUND: "見つかった {0} {チャンク|チャンク} 変更します。"
-ROLLBACK_CHUNKS_MODIFIED: "変更 {0}/{1} {チャンク|チャンク}。"
-ROLLBACK_COMPLETED: "{ロールバック|復元|プレビュー} のために完成{0}\" 。"
-ROLLBACK_EXCLUDED_USERS: "除外 {ユーザー|ユーザー}: \"{0}\" 。"
-ROLLBACK_INCLUDE: "{含まれる|除外される} {ブロック|エンティティ|ターゲット} {タイプ|タイプ}: \"{0}\" 。"
+ROLLBACK_CHUNKS_FOUND: "変更する{0}{チャンク|チャンク}が見つかりました。"
+ROLLBACK_CHUNKS_MODIFIED: "{0}/{1}{チャンク|チャンク} 変更"
+ROLLBACK_COMPLETED: "{ロールバック|復元|プレビュー} は\"{0}\"で完了しました。"
+ROLLBACK_EXCLUDED_USERS: "除外するユーザー: \"{0}\" "
+ROLLBACK_INCLUDE: "{含まれる|除外される}{ブロック|エンティティ|ターゲット} タイプ: \"{0}\""
 ROLLBACK_IN_PROGRESS: "ロールバック/復元はすでに進行中です。"
-ROLLBACK_LENGTH: "所要時間:{0} {秒|秒}。"
-ROLLBACK_MODIFIED: "{変更|変更} {0}。"
-ROLLBACK_RADIUS: "半径:{0} {ブロック|ブロック}。"
-ROLLBACK_SELECTION: "半径を「{0}\" 。"
-ROLLBACK_STARTED: "{ロールバック|復元|プレビュー} 「で始まった{0}\" 。"
-ROLLBACK_TIME: "時間範囲:{0}。"
+ROLLBACK_LENGTH: "所要時間:{0} {秒|秒}"
+ROLLBACK_MODIFIED: "{変更|変更} {0}"
+ROLLBACK_RADIUS: "半径:{0} ブロック"
+ROLLBACK_SELECTION: "半径を\"{0}\"にします。"
+ROLLBACK_STARTED: "{ロールバック|復元|プレビュー}が「{0}」で開始されました。"
+ROLLBACK_TIME: "時間範囲: {0}"
 ROLLBACK_WORLD_ACTION: "{0}{世界|アクション}に制限があります。"
-SIGN_HEADER: "メッセージに署名する"
-STATUS_CONSUMER: "消費者: {0} {item|items} 順番待ち。"
+SIGN_HEADER: "メッセージに署名"
+STATUS_CONSUMER: "消費者: {0}{アイテム|アイテム達}のキューを待っています"
```

変更漏れ
omission of change

```suggestion
STATUS_CONSUMER: "{0}{アイテム|アイテム達}のキューを処理中"
```

消費中はコンシューム中と言ったりしますが専門用語すぎるので処理中に直してみました。
While consuming, it is also called consume, but that is too technical a term, so I changed it to process.